pmd-apex/src/main/resources/category/apex/design.xml

```diff
@@ -35,6 +35,32 @@ public class Foo {
         </example>
     </rule>
 
+    <rule name="AvoidUnusedMethod"
+          since="5.5.0"
+          message="Deeply nested if..then statements are hard to read"
```

Comment:
The metadata of this rule needs to be updated (since, message, description, example)

Reply:
Done